test/screenshot/lib/cli-arg-parser.js

```diff
@@ -114,10 +147,167 @@ E.g., 'origin/master' (default), 'HEAD', 'feat/foo/bar', 'fad7ed3'.
     return (this.args_['mdc_exclude_browser'] || []).map((pattern) => new RegExp(pattern));
   }
 
+  /** @return {string} */
+  get testDir() {
+    return this.args_['mdc_test_dir'];
+  }
+
+  /** @return {string} */
+  get goldenPath() {
+    return this.args_['mdc_golden_path'];
+  }
+
   /** @return {string} */
   get diffBase() {
     return this.args_['mdc_diff_base'];
   }
+
+  /**
+   * @param {string} rawDiffBase
+   * @param {string} defaultGoldenPath
+   * @return {!Promise<!DiffSource>}
+   */
+  async parseDiffBase({
```

Comment:
I was worried this function would be complex given all the different types of inputs but it's quite clear.

Reply:
Yay!! Glad to hear it 😀